x-pack/plugins/encrypted_saved_objects/server/plugin.ts

```diff
@@ -26,8 +25,8 @@ export interface PluginsSetup {
 }
 
 export interface EncryptedSavedObjectsPluginSetup {
+  canEncrypt: boolean;
```

Comment:
**optional** It's probably a good idea to start documenting our public interface

```suggestion
  /** Indicates if Saved Object encryption is possible. Requires an encryption key to be explicitly set via `xpack.encryptedSavedObjects.encryptionKey`. */
  canEncrypt: boolean;
```

Reply:
Definitely agree, thanks!